### PR TITLE
Added UC11

### PR DESCRIPTION
**Pull request by Milesight-IoT**
#### Summary

- Added UC1114, UC1122, UC1152

#### Changes

- Added discontinued UC1114, UC1122, UC1152
- Removed productURL, datasheetURL, resellerURL in uc1114.yaml, uc1122.yaml, uc1152.yaml because Milesight IoT no longer provide the datasheet and purchasing service for UC1114, UC1122, UC1152
- Updated index.yaml

#### Notes for Reviewers

- productURL, datasheetURL, resellerURL are removed according to advice from @johanstokking .
- @Jaime-Trinidad will take over to review. 

#### Release Notes

- ...

### CODE REVIEW

**Comment by Jaime-Trinidad** [comment]
Hi @Milesight-IoT there are 80 conflicts to resolve can you check them, also the firmware version is recommended to add the new one, no overwrite the older one, this may affect existing devices. Please check on it so I make another review

**Comment by Milesight-IoT** [comment]
Hi @Jaime-Trinidad could you please share hints about this error?
`invalid index: .endDevices[0] should match pattern "^[a-z0-9](?:[-]?[a-z0-9]){2,}$"`

**Comment by Jaime-Trinidad** [comment]
> Hi @Jaime-Trinidad could you please share hints about this error? `invalid index: .endDevices[0] should match pattern "^[a-z0-9](?:[-]?[a-z0-9]){2,}$"`

may refer that you are using Capital letters, or maybe leave a space, can you give me a better context, which file is the one?

**Comment by johanstokking** [comment]
> Hi @Jaime-Trinidad could you please share hints about this error? `invalid index: .endDevices[0] should match pattern "^[a-z0-9](?:[-]?[a-z0-9]){2,}$"`

Where do you see that error?

The CI here reports this error:

```
milesight-iot: am319: EU863-870: codec am319-codec invalid: .uplinkDecoder.examples[0].input.bytes[12] should be integer
```

File: vendor/milesight-iot/am319-codec.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Uplink decoder decodes binary data uplink into a JSON object
+uplinkDecoder:
+  fileName: am319.js
+
+  examples:
+    - description: Ambience Monitoring Sensors (example 1) - Milesight IoT
+      input:
+        fPort: 85
+        bytes:
+          [0x03,0x67,0x34,0x01,0x04, 0x68, 0x65,0x05,0x00,0x01,0x06,0xCB, 0x02, 0x07, 0x7D,0xA8,0x03,0x08,0x7D,0x25,0x00, 0x09,0x73,0x66,0x27, 0x0A, 0x7D,0x04,0x00,0x0B,0x7D,0x20,0x00, 0x0C, 0x7D, 0x30, 0x00 ]
```
Comment (johanstokking) [inline vendor/milesight-iot/am319-codec.yaml:10]: To be honest, I'm not sure if we support hexadecimal here? @Jaime-Trinidad

**Comment by Milesight-IoT** [comment]
Hi @johanstokking , do you have time to review our new changes?

**Comment by johanstokking** [comment]
@Milesight-IoT please rebase on master and make sure only files in the Milesight folder are affected. See https://github.com/TheThingsNetwork/lorawan-devices/pull/490/files.

**Pull request closed without merging**